it/mail/chrome/messenger/imapMsgs.properties

```diff
@@ -365,107 +365,107 @@
-5111=The IMAP server %S does not allow plaintext passwords. Please try changing to 'Encrypted password' as the 'Authentication method' in the 'Account Settings | Server settings'.
+5111=Il server IMAP %S non consente password in testo semplice. Seleziona 'Password crittografate' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'.
 
 ## @name IMAP_AUTH_CHANGE_ENCRYPT_TO_PLAIN_NO_SSL
 ## @loc None
 # LOCALIZATION NOTE (5112): %S is the server hostname
-5112=The IMAP server %S does not seem to support encrypted passwords. If you just set up the account, please try changing to 'Password, transmitted insecurely' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, this is a common scenario how someone could steal your password.
+5112=Sembra che il server IMAP %S non supporti le password crittografate. Se hai appena impostato questo account, prova a selezionare 'Password, transmesse in maniera non sicura' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'. Se di solito funzionava e ora improvvisamente non funziona, questa è una situazione comune (qualcuno potrebbe rubarti la password).
 
 ## @name IMAP_AUTH_MECH_NOT_SUPPORTED
 ## @loc None
 # LOCALIZATION NOTE (5113): %S is the server hostname
-5113=The IMAP server %S does not support the selected authentication method. Please change the 'Authentication method' in the 'Account Settings | Server settings'.
+5113=Il server IMAP %S non supporta il metodo di autenticazione selezionato. Modifica il 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'.
 
 ## @name IMAP_AUTH_MECH_FAILED
 ## @loc None
 # LOCALIZATION NOTE (5114): %S is the server hostname
-5114=All login mechanisms for %S failed. Please check the password or change the 'Authentication method' in the 'Account Settings | Server settings'.
+5114=Nessuna procedura di accesso %S è riuscita. Controlla la password o modifica il 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'.
 
 ## @name IMAP_AUTH_GSSAPI_FAILED
 ## @loc None
 # LOCALIZATION NOTE (5115): %S is the server hostname
-5115=The Kerberos/GSSAPI ticket was not accepted by the IMAP server %S. Please check that you are logged in to the Kerberos/GSSAPI realm.
+5115=Il ticket Kerberos/GSSAPI non è stato accettato dal server IMAP %S. Controlla di aver effettuato l'accesso al reame Kerberos/GSSAPI.
 
 ## @name IMAP_LOGIN_DISABLED
 ## @loc None
 # LOCALIZATION NOTE (5116): %S is the account name
-5116=You cannot log in to %S because the server doesn't allow plaintext authentication without STARTTLS or SSL/TLS. Try enabling connection security or secure authentication in the account settings.
+5116=Non puoi accedere a %S perché il server non consente autenticazione con testo semplice senza STARTTLS o SSL/TLS. Prova ad attivare la sicurezza connessione o l'autenticazione sicura nelle impostazioni account.
```

Comment:
CORRECTIONS NEEDED. 

An article is missing. Please add "l'" before "autenticazione con testo semplice".